app/models/preserved_object.rb

```diff
@@ -19,8 +19,8 @@ class PreservedObject < ApplicationRecord
 
   # Spawn asynchronous checks of all existing archive complete_moabs.
   # This logic is similar to PlexerJob, for a different purpose.
-  # This should implement the start of the replication process if status is unreplicated for an archival pres_copy
-  # Compare last_existence_check (from archive pres_copy) with archive TTL when checking the archival pres_copy status
+  # This should implement the start of the replication process if status is unreplicated for an archival comp_moab
```

Comment:
i think with the new terminology and data model, a more accurate sentence might be:  `This should implement the start of the replication process if the CompleteMoab doesn't have all of the expected ZippedMoabVersion objects in a fully replicated state.`